config/locales/fr.yml

```diff
@@ -0,0 +1,25 @@
+fr:
+  refinery:
+    plugins:
+      refinerycms_copywriting:
+        title: Copywriting
+        title_tab: Copywriting
+    copywriting:
+      admin:
+        phrases:
+          locale_filters:
+            change_language: Changer la langue
+          scope_filters:
+            filter_by_scope: Filtrer par contexte
+            filtered_by: "Filtré par '%{filter_scope}'"
+            clear_scope_filter: Supprimer le filtre
+          records:
+            no_items_yet: Il n'y a pas de phrases de copywriting.
```

Comment:
I changed it for a "nom propre" and referred to the plugin.
I have try to find a good french translation for Copywriting but it sounds weird.